### DISCUSSION TITLE
Pywebview ?

### DISCUSSION THREAD

**Discussion by imxade** [beeware/briefcase#1964] [Q&A] [answered]
Can GUI created with `pywebview` be packaged ?

**Comment by freakboy3742** [comment, marked as answer]
Briefcase should be able to package *any* GUI framework, as long as that GUI framework can be installed entirely from PyPI.

I've never used Pywebview before, but I was able to get a "hello world" example running by:
* Installing Briefcase
* Running `briefcase new`, selecting "No GUI framework" from the wizard
* adding `requires = ["pywebview"]` to the generated pyproject.toml
* Adding the "hello world" tutorial code from the pywebview website to the generated app.py's main method.
